pkg/k8s/job.go

```
@@ -191,3 +180,37 @@ func DeleteK8sJob(clientset *kubernetes.Clientset, namespace string, jobName str
 	jobs := clientset.BatchV1().Jobs(namespace)
 	return jobs.Delete(context.TODO(), jobName, metav1.DeleteOptions{})
 }
+
+func prepareJobEnv(task config.Task, eventData *keptnv2.EventData, jsonEventData interface{}) []v1.EnvVar {
+
+	var jobEnv []v1.EnvVar
+	for _, env := range task.Env {
+		value, err := jsonpath.Get(env.Value, jsonEventData)
+		if err != nil {
+			log.Printf("Could not add env with name %v, value %v, skipping: %v", env.Name, env.Value, err)
+			continue
```

Can we return the error here? This indicates a misonfiguration, which should cause the service run to fail -> fail early.